decompiler/config/all-types.gc

```diff
@@ -10312,19 +10311,20 @@
 
 ;; - Types
 
+;; NOTE - root can also be a collide-shape
 (deftype manipy (process-drawable)
-  ((new-trans-hook       function        :offset-assert 176)
-   (cur-trans-hook       function        :offset-assert 180)
-   (cur-event-hook       function        :offset-assert 184)
+  ((new-trans-hook       (function none)        :offset-assert 176) ;; TODO - no idea on these types
+   (cur-trans-hook       (function none)        :offset-assert 180) ;; TODO - no idea on these types
+   (cur-event-hook       (function none)        :offset-assert 184) ;; TODO - no idea on these types
```

Comment:
probably should return `object` but ok for now. I'd guess these are state handlers